### DISCUSSION TITLE
Easy tasks for new-comers?

### DISCUSSION THREAD

**Discussion by sanket1729** [rust-bitcoin/rust-miniscript#636] [General]
I presume this Summer of Bitcoin season and many new contributors want to participate to contribute to new issues to make a strong application. I may not have time to participate this year as a mentor. What can be done to keep the maintainer/reviewer burden at minimum? 

 I presume rust bitcoin cookbook can be a good starter place? I presume there maybe a some good tasks there. @apoelstra @tcharding

**Comment by apoelstra** [comment]
I think the psbtv2 crate might be a good place to start (cc @tcharding) and updating rust-miniscript to use that, once it's ready.

**Comment by tcharding** [comment]
I'll review in psbtv2 but I currently personally maintain a pretty hard standard on new devs because they tend to do one patch and disappear - I'm definitely not put on this planet to help down right beginner devs get a single patch into open source projects :) Sorry to be so harsh, I have not enjoyed the Summer of Bitcoin experience the last two years.

**Reply by sanket1729** [reply]
Sorry for your experience. I also don't like being bombarded by new comments/low quality PRs during this season. We can continue answering questions on open issues as people ask them. I will ask Summer of Orgranizer to not ask to open issues if this is first time they are programming either rust or bitcoin.

**Reply by tcharding** [reply]
Legend, thanks.
